d3a_api_client/utils.py

```diff
@@ -41,3 +41,31 @@ def get_request(endpoint, data, jwt_token):
                       f"Response body: {resp.text}")
         return False
     return True
+
+
+def get_area_uuid_from_area_name(serialized_scenario, area_name):
+    if "name" in serialized_scenario and serialized_scenario["name"] == area_name:
+        return serialized_scenario["uuid"]
+    if "children" in serialized_scenario:
+        for child in serialized_scenario["children"]:
+            area_uuid = get_area_uuid_from_area_name(child, area_name)
+            if area_uuid is not None:
+                return area_uuid
+    return None
+
+
+def get_area_uuid_from_area_name_and_collaboration_id(collab_id, area_name, domain_name):
+    jwt_key = retrieve_jwt_key_from_server(domain_name)
+    from sgqlc.endpoint.http import HTTPEndpoint
+
+    url = f"{domain_name}/graphql/"
+    headers = {'Authorization': f'JWT {jwt_key}', 'Content-Type': 'application/json'}
+
+    query = 'query { readConfiguration(uuid: "{' + collab_id + \
+            '}") { scenarioData { representation { serialized } } } }'
+
+    endpoint = HTTPEndpoint(url, headers)
+    data = endpoint(query=query)
+    return get_area_uuid_from_area_name(
+        json.loads(data["data"]["readConfiguration"]["scenarioData"]["representation"]["serialized"]), area_name
+    )
```

Comment:
It could be the case where no area_uuid is catched with provided `area_name` parameter, if Im right then an additional error message should be thrown IMO ?

Reply:
ATM None is returned from the function at the case you mention. That way the user of the function knows that no area was found implicitly, by checking the return value and finding out that it is None. Therefore functionally it works as it is now. However, if you prefer an exception instead of the None return value please write it here and I'll adapt it.

Reply:
The reason Im insisting it a bit more is since it is client end that may need to know to debug what went wrong. Even a small message like `No area exist with this name inside Collaboration.` would be nice to have.

Reply:
Persuaded. Adding the exception, thanks!